tikv/deploy-tikv-using-ansible.md

```diff
@@ -0,0 +1,338 @@
+    
+    > **Note:** When you deploy TiKV using Ansible, use SSD disks for the data directory of TiKV and PD nodes.
+
+2. A Control Machine that meets the following requirements:
+
+    > **Note:** The Control Machine can be one of the target machines.
+    
+    - CentOS 7.3 (64 bit) or later with Python 2.7 installed
+    - Access to the Internet
+    - Git installed
+    - SSH Mutual Trust configured
+
+        In the Control Machine, you can log in to the deployment target machine using `tidb` user account without a password. See [How to configure SSH mutual trust and sudo without password](../op-guide/ansible-deployment#how-to-configure-ssh-mutual-trust-and-sudo-without-password).
+
+## Step 1: Download TiDB-Ansible to the Control Machine
+
+1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory.
+
+2. Download the corresponding TiDB-Ansible version. The default folder name is `tidb-ansible`.
+
+    - Download the 2.0 GA version:
+
+        ```bash
+        git clone -b release-2.0 https://github.com/pingcap/tidb-ansible.git
+        ```
+    
+    - Download the master version:
+
+        ```bash
+        git clone https://github.com/pingcap/tidb-ansible.git
+        ```
+
+    You can turn to the official team (info@pingcap.com) for advice on which version to choose.
+
+## Step 2: Install Ansible and dependencies in the Control Machine
+
+1. Install Ansible and dependencies on the Control Machine:
+
+    ```bash
+    sudo yum -y install epel-release
+    sudo yum -y install python-pip curl
+    cd tidb-ansible
+    sudo pip install -r ./requirements.txt
+    ```
+
+    Ansible and related dependencies are recorded in the `tidb-ansible/requirements.txt` file.
+
+2. View the version of Ansible:
+
+    ```bash
+    ansible --version
+    ```
+
+    Currently, the 1.0 GA version depends on Ansible 2.4, while the 2.0 GA version and the master version are compatible with Ansible 2.4 and Ansible 2.5.
+
+## Step 3: Edit the `inventory.ini` file to orchestrate the TiKV cluster
+
+Edit the `tidb-ansible/inventory.ini` file to orchestrate the TiKV cluster. The standard TiKV cluster contains 6 machines: 3 PD nodes and 3 TiKV nodes.
+
+- Deploy at least 3 instances for TiKV.
+- Do not deploy TiKV together with PD on the same machine.
+- Use the first PD machine as the monitoring machine.
+
+> **Note:**
+>
+> - Leave `[tidb_servers]` in the `inventory.ini` file empty, because this deployment is for the TiKV cluster, not the TiDB cluster.
+> - It is required to use the internal IP address to deploy.
+
+You can choose one of the following two types of cluster topology according to your need:
+
+- [The cluster topology of a single TiKV instance on each TiKV node](#option-1-use-the-cluster-topology-of-a-single-tikv-instance-on-each-tikv-node)
+
+    By default, deploy one TiKV instance on each TiKV node.
```

Comment:
-> In most cases, it is recommended to deploy one TiKV instance on each TiKV node for better performance. However, if the CPU and memory of your TiKV machines are much better than the required in [Hardware and Software Requirements](../op-guide/recommendation.md), and you have more than two disks in one node or the capacity of one SSD is larger than 2 T, you can deploy no more than 2 TiKV instances on a single TiKV node.